framework/tlvf/src/src/CmduTlvParser.cpp

```diff
@@ -108,7 +108,7 @@ std::shared_ptr<BaseClass> CmduTlvParser::parseNextTlv()
     case (17): {
         auto tlv = cmdu_.addClass<ieee1905_1::tlvWsc>();
         if (tlv)
-            WSC::AttrList::parse(*tlv);
+            tlv->addInnerClassList(WSC::AttrList::parse(*tlv));
         return tlv;
     }
```

Comment:
CmduTlvParser shouldn't handle special cases such as this one, the attribute list parsing should be done as part of `tlvWsc::addClass()`, this requires a TLVF change, as tlvWsc is an autogenerated class, and I will create an issue for that.
But still worth mentioning.